playbook-website/app/javascript/components/AvailableProps/kitProps.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react'
 import { Body, Card, Table, Title } from 'playbook-ui'
 
-const KitProps = ({ kitPropsValues, darkMode }) => {
+type KitPropsType = {
+  kitPropsValues: {[key: string]: any},
+  darkMode: boolean
+}
+
+const KitProps = ({ kitPropsValues, darkMode }: KitPropsType) => {
+  const getTypeName = (typeName: string) => {
+    if (typeName.indexOf('{') !== -1) {
+      return 'object'
+    }
+    if (typeName.indexOf('|') !== -1) {
+      return 'union'
+    }
+    if (typeName.indexOf('=>') !== -1) {
+      return 'function'
+    }
+    return typeName
+  }
```

Comment:
Here, we are aliasing some types intentionally in a naive way to help with readability.